src/me/StevenLawson/TotalFreedomMod/Commands/Command_kick.java

```diff
@@ -0,0 +1,45 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_PlayerList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
+@CommandParameters(description = "For the bad OPs", usage = "/<command> <playername>")
+public class Command_kick extends TFM_Command
+{
+    @Override
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (args.length > 1)
+        {
+            return false;
+        }
+        
+        Player player = bukkit.getServer().getPlayer(args[0]);
+        
+        if (player == null)
+        {
+            sender.sendMessage(TFM_Command.PLAYER_NOT_FOUND);
+            return true;
+        }
+        
+        String reason = StringUtils.join(ArrayUtils.subarray(args, 1, args.length), " ");
+        if (reason == null) {
+            TFM_Util.adminAction(sender.getName(), "Kicking " + player.getName() + " from the server", true);
```

Comment:
Instead of having this twice, once in the 'if' and once in the 'else', please move these lines out of the if statement.